research/sentiment_analysis/sentiment_main.py

```diff
@@ -85,8 +101,11 @@ def run_model(dataset_name, emb_dim, voc_size, sen_len,
                       help="The number of epochs for training.",
                       type=int, default=55)
 
+  parser.add_argument("-f", "--folder",
+                      help="folder/dir to save trained model",
+                      type=str, default=os.getcwd())
```

Comment:
default None and raise error?

Reply:
updated, instead of raising error i called parser.error so it will take care of raising error with message passed to it.